int16 modID
int16 modulehealth
time stamp
